From b016760f3b81ba91719c7ecbf0d31045f90775e4 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Tue, 27 Jan 2015 17:26:48 +0800
Subject: [PATCH] MLK-10101-5 usb: chipidea: otg: init otg_fsm of usb_bus
 while start host

Set otg_fsm of usb_bus to be ci->fsm while start host.

Signed-off-by: Li Jun <jun.li@freescale.com>
---
 drivers/usb/chipidea/host.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/chipidea/host.c b/drivers/usb/chipidea/host.c
index 40131e4..85febf7 100644
--- a/drivers/usb/chipidea/host.c
+++ b/drivers/usb/chipidea/host.c
@@ -264,6 +264,7 @@ static int host_start(struct ci_hdrc *ci)
 		ci->hcd = hcd;
 
 		if (ci_otg_is_fsm_mode(ci)) {
+			hcd->self.otg_fsm = &ci->fsm;
 			otg->host = &hcd->self;
 			hcd->self.otg_port = 1;
 		}
-- 
1.8.0

